src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -67,6 +67,7 @@ extern std::set<ScriptEditorImplementation *> g_pScriptEditorWindowList;
 extern KviModule * g_pEditorModulePointer;
 
 static QColor g_clrBackground(0, 0, 0);
+static QColor g_clrBackgroundHighlight(50, 50, 50);
```

Comment:
You can easily disable it by setting both background colors to be the same.  I fail to see the need for a toggle.